front/components/welcome/WelcomeSecond.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-content>
+    <welcome-desktop
+      class="hidden-mobile"
+      :params="params"
+      @next="emitNext"
+      @mark-as-welcomed="emitMarkAsWelcomed"
+      @left-clicked="emitLeftClicked"
+      @right-clicked="emitRightClicked"
+    ></welcome-desktop>
+    <welcome-mobile-portrait
+      class="hidden-desktop hidden-landscape"
+      :params="params"
+      :hide-next-btn="true"
+      @mark-as-welcomed="emitMarkAsWelcomed"
+      @right-clicked="emitRightClicked"
+      @left-clicked="emitLeftClicked"
+    ></welcome-mobile-portrait>
+    <welcome-mobile-landscape
+      class="hidden-desktop hidden-portrait"
+      :params="params"
+      :hide-next-btn="true"
```

Comment:
dont need  to pass true if its boolena, just prop enough

Reply:
Not sure exactly what your suggestion is here. Do you suggest I replace `:hide-next-btn="true"` with `hide-next-btn=""` or something else? If so, why remove the `:`?

Reply:
:hide-next-btn="true"  equals hide-next-btn without : or any params

Reply:
The change has been made

Reply:
dont see change xD